### ISSUE TITLE
How to get the quality scores

### ISSUE THREAD

**Issue by ver0z** [implus/GFocal#32]
Can I get the quality scores only with the logits ?
